recommender/RecommenderTest.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+import unittest
+import os
+from recommender import *
+
+class RecommenderTest(unittest.TestCase):
+    def testEnvironmentVariables(self):
+        self.assertNotEqual(os.environ.get('GOOGLE_API_KEY'), None)
+        self.assertNotEqual(os.environ.get('OMNIBUS_DATABASE_URL'), None)
+
+    def testInitialization(self):
+        recommender = Recommender("person1", datetime.date.today(), ["keyword1", "keyword2", "keyword3"])
+        self.assertEqual(recommender._userID, "person1")
+        self.assertEqual(recommender._date, datetime.date.today())
```

Comment:
Nitpicking: Normally don't add datetime.* to the unit tests since they are scenarios where it will fail due to timing.